.gitignore

```diff
@@ -2,3 +2,4 @@ data/
 .idea
 .env
 certs/
+/chainlink_config/tempkeys
```

Comment:
removing these keys ( - I think, didn't test ist recently - ) might result in the chainlink node creating new ones, so it probably will not work with the preloaded addresses on the sidechain. The chainlink node wont have any funds on its new address and then can't create transactions. Unless these keys are also in the database file, shared to the chainlink container, then it might still work without these key files.
The crosschain chainlink e2e workflow needs to be tested without these files before removing them, I will try to do this today before my holiday...

Reply:
@samt1803 This isn't urgent, just like the title says